docs/source-pytorch/common/lightning_module.rst

```diff
@@ -1052,6 +1052,51 @@ Set and access example_input_array, which basically represents a single batch.
         # generate some images using the example_input_array
         gen_images = self.generator(self.example_input_array)
 
+truncated_bptt_steps
```

Comment:
This is actually not a LightningModule attribute and we shouldn't treat it as such.

In my opinion this fits more one of the how to guides https://github.com/Lightning-AI/pytorch-lightning/tree/master/docs/source-pytorch/common (they end up in the How to guides sections in the docs) where you could also have more room to expand on what TBPTT is and maybe have a figure explaining the concept.

Reply:
which doc under `source-pytorch/common` do you think this explanation below to if I may ask?

Reply:
I would create a new `tbptt.rst` doc and add details there.
It doesn't need to be large, you can model it after https://github.com/Lightning-AI/pytorch-lightning/blob/master/docs/source-pytorch/common/early_stopping.rst

Reply:
ack